diff --git a/node_modules/react-native-swiper-flatlist/src/components/SwiperFlatList/SwiperFlatList.tsx b/node_modules/react-native-swiper-flatlist/src/components/SwiperFlatList/SwiperFlatList.tsx
index ecc4ae8..cd81c8a 100644
--- a/node_modules/react-native-swiper-flatlist/src/components/SwiperFlatList/SwiperFlatList.tsx
+++ b/node_modules/react-native-swiper-flatlist/src/components/SwiperFlatList/SwiperFlatList.tsx
@@ -16,7 +16,7 @@ const ITEM_VISIBLE_PERCENT_THRESHOLD = 60;
 
 // TODO: figure out how to use forwardRef with generics
 type T1 = any;
-type ScrollToIndex = { index: number; animated?: boolean };
+type ScrollToIndex = { index: number; animated?: boolean, viewOffset?: number };
 
 // const SwiperFlatList = React.forwardRef<RefProps, SwiperFlatListProps<SwiperType>>(
 export const SwiperFlatList = React.forwardRef(
@@ -94,8 +94,8 @@ export const SwiperFlatList = React.forwardRef(
 
     const _scrollToIndex = React.useCallback(
       (params: ScrollToIndex) => {
-        const { index: indexToScroll, animated = true } = params;
-        const newParams = { animated, index: indexToScroll };
+        const { index: indexToScroll, animated = true, viewOffset } = params;
+        const newParams = { animated, index: indexToScroll, viewOffset };
 
         setIgnoreOnMomentumScrollEnd(true);
 
